core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.css

```diff
@@ -123,6 +129,12 @@
   stroke-width: 2px;
 }
 
+#dag-viz-graph svg.stage g.cluster.barrier rect {
+  fill: #84E9E2;
```

Comment:
Just one minor question, how is the color decided?  Though I don't have a better idea either.

Reply:
I chose the color from [here](https://colorhunt.co).
I considered the base color for barrier-exec mode should match with existing color (blue-based colors) and avoided red-based colors because they remind us of error or warning.

Reply:
I see. Thanks for the explanation!